ledger/ledger-api-client/src/main/scala/com/digitalasset/ledger/client/services/version/withoutledgerid/VersionClient.scala

```diff
@@ -22,4 +24,37 @@ private[daml] final class VersionClient(service: VersionServiceStub) {
         new GetLedgerApiVersionRequest(ledgerIdToUse.unwrap)
       )
       .map(_.version)
+
+  def getApiFeatures(
+                     ledgerIdToUse: LedgerId,
+                     token: Option[String] = None,
+                   )(implicit executionContext: ExecutionContext): Future[Seq[VersionClient.Feature]] =
+    LedgerClient
+      .stub(service, token)
+      .getLedgerApiVersion(
+        new GetLedgerApiVersionRequest(ledgerIdToUse.unwrap)
+      )
+      .map(_.features.toSeq.flatMap(VersionClient.Feature.fromProto))
+}
+
+@SuppressWarnings(Array("org.wartremover.warts.Option2Iterable"))
+private[daml] object VersionClient {
+  sealed trait Feature
+  object Feature {
```

Comment:
Usually we define these datatypes in the domain module, e.g., here for user rights https://github.com/digital-asset/daml/blob/f057765cd27ecefcdf903ab92a8b340357cdb5c6/ledger/ledger-api-domain/src/main/scala/com/digitalasset/ledger/api/domain.scala#L403

Reply:
Thanks, will move it.